### PR TITLE
(AzureCXP) fixes MicrosoftDocs/azure-docs#106119

### PR DESCRIPTION
**Pull request by mumurug-MSFT**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.logic/logic-app-create/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ languages:
...
 
-This quickstart template creates a basic [Consumption logic app workflow](https://docs.microsoft.com/azure/logic-apps/logic-apps-overview#resource-type-differences) that runs in multi-tenant Azure Logic Apps and connects to the status update feed for Azure services at https://status.azure.com/status/feed, which is displayed at https://status.azure.com/status.
+This quickstart template creates a basic [Consumption logic app workflow](https://docs.microsoft.com/azure/logic-apps/logic-apps-overview#resource-type-differences) that runs in multi-tenant Azure Logic Apps and connects to the status update feed for Azure services at https://azure.status.microsoft/status/feed, which is displayed at https://azure.status.microsoft/status.
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/README.md:31]: Just noticed the doc URL has changed:

```suggestion
This quickstart template creates a basic [Consumption logic app workflow](https://learn.microsoft.com/azure/logic-apps/logic-apps-overview#resource-environment-differences) that runs in multi-tenant Azure Logic Apps and connects to the status update feed for Azure services at https://azure.status.microsoft/status/feed, which is displayed at https://azure.status.microsoft/status.
```

File: quickstarts/microsoft.logic/logic-app-create/README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ languages:
 
 Azure Logic Apps is a cloud service that automates the execution of your business processes. You can create a workflow by using a visual designer to arrange prebuilt components into the sequence that you need. When you save your workflow, the designer sends the workflow's definition to the Azure Logic Apps execution engine. When the conditions for the workflow's trigger are met, the engine launches the workflow and manages the compute resources that the workflow needs to run. If you're new to Azure Logic Apps, see [What is Azure Logic Apps?](https://docs.microsoft.com/azure/logic-apps/logic-apps-overview).
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/README.md:29]: Doc domain has changed:

```suggestion
Azure Logic Apps is a cloud service that automates the execution of your business processes. You can create a workflow by using a visual designer to arrange prebuilt components into the sequence that you need. When you save your workflow, the designer sends the workflow's definition to the Azure Logic Apps execution engine. When the conditions for the workflow's trigger are met, the engine launches the workflow and manages the compute resources that the workflow needs to run. If you're new to Azure Logic Apps, see [What is Azure Logic Apps?](https://learn.microsoft.com/azure/logic-apps/logic-apps-overview).
```

File: quickstarts/microsoft.logic/logic-app-create/README.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ languages:
...
 
-This quickstart template creates a basic [Consumption logic app workflow](https://docs.microsoft.com/azure/logic-apps/logic-apps-overview#resource-type-differences) that runs in multi-tenant Azure Logic Apps and connects to the status update feed for Azure services at https://status.azure.com/status/feed, which is displayed at https://status.azure.com/status.
+This quickstart template creates a basic [Consumption logic app workflow](https://docs.microsoft.com/azure/logic-apps/logic-apps-overview#resource-type-differences) that runs in multi-tenant Azure Logic Apps and connects to the status update feed for Azure services at https://azure.status.microsoft/status/feed, which is displayed at https://azure.status.microsoft/status.
 
 For information about using this template, see [Create Azure Resource Manager templates for Azure Logic Apps](https://docs.microsoft.com/azure/logic-apps/logic-apps-create-deploy-template). To learn more about how to deploy the template, see the [quickstart](https://docs.microsoft.com/azure/logic-apps/quickstart-create-deploy-azure-resource-manager-template) article.
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/README.md:33]: Doc domain has changed:

```suggestion
For information about using this template, see [Create Azure Resource Manager templates for Azure Logic Apps](https://learn.microsoft.com/azure/logic-apps/logic-apps-create-deploy-template). To learn more about how to deploy the template, see the [quickstart article](https://learn.microsoft.com/azure/logic-apps/quickstart-create-deploy-azure-resource-manager-template).
```

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

File: quickstarts/microsoft.logic/logic-app-create/README.md [hunk 4]
```diff
@@ -26,11 +26,11 @@ languages:
...
 
-For information about using this template, see [Create Azure Resource Manager templates for Azure Logic Apps](https://docs.microsoft.com/azure/logic-apps/logic-apps-create-deploy-template). To learn more about how to deploy the template, see the [quickstart](https://docs.microsoft.com/azure/logic-apps/quickstart-create-deploy-azure-resource-manager-template) article.
+For information about using this template, see [Create Azure Resource Manager templates for Azure Logic Apps](https://learn.microsoft.com/azure/logic-apps/logic-apps-create-deploy-template). To learn more about how to deploy the template, see the [quickstart article](https://learn.microsoft.com/azure/logic-apps/quickstart-create-deploy-azure-resource-manager-template).
 
 - [Azure Logic Apps](https://azure.microsoft.com/services/logic-apps/)
 - [Azure Logic Apps documentation](https://docs.microsoft.com/azure/logic-apps/logic-apps-overview)
```
Comment (ecfan) [inline quickstarts/microsoft.logic/logic-app-create/README.md:36]: Sorry, didn't see these earlier:

```suggestion
- [Azure Logic Apps documentation](https://learn.microsoft.com/azure/logic-apps/logic-apps-overview)
```

**Comment by azure-quickstarts** [comment]
@ecfan - check this PR for updates that may be needed to documentation that references this sample.  [This is an automated message. You are receiving it because you are listed as the docOwner in metadata.json.]

**Review by ecfan — APPROVED** [review]
@mumurug-MSFT: Thank you!

**Pull request merged**